glpk/bld.bat

```diff
@@ -0,0 +1,14 @@
+cd w%ARCH%
+copy config_VC config.h
+
+nmake /f Makefile_VC
+
+mkdir %PREFIX%\bin

```

Comment:
We're having problems with this recipe on Windows and I think it's because it's copying the binary to the wrong location. See http://conda.pydata.org/docs/building/environment-vars.html for the value of `LIBRARY_BIN` and https://github.com/snorfalorpagus/conda-recipes/blob/116af80/glpk/bld.bat#L5 for a recipe that works.